client/src/plus/index.scss

```diff
@@ -50,12 +55,11 @@
         flex-direction: column;
         justify-content: center;
         align-items: center;
-        padding: 64px 32px;
         overflow: hidden;
-        box-shadow: 0px 0px 7px rgba(0, 0, 0, 0.25);
+        box-shadow: 0px 0px 7px rgba(0, 0, 0, 0.548);
 
-        @media #{$mq-tablet-and-up} {
-          padding: 64px;
+        &:focus {
+          outline: none;
         }
```

Comment:
I am a little concerned about this one but, as long as the user can clearly see where focus moves once the modal is open and they press the tab key, this should be ok.

Reply:
Yeah I share that worry, I was not sure what the best course of action is here since we do want to trap focus in the modal and have it go to the next inner modal element, but the modal being focused is really just incidental to that and not anything meaningful